dist/css/bootstrap.css

```diff
@@ -3987,7 +3987,7 @@ input[type="button"].btn-block {
 
 .card-header {
   padding: 0.75rem 1.25rem;
-  background-color: #f5f5f5;
+  background-color: whitesmoke;
```

Comment:
sass compiled that to colournames, not me. Anywho that's in the compiled css, not in the scss.